src/node/tests/elder.rs

```diff
@@ -219,7 +219,7 @@ impl Env {
         new_info: &DkgToSectionInfo,
     ) -> Result<()> {
         self.cast_unordered_votes(
-            self.quorum_count() - 1,
+            quorum_count(new_info.new_elders_info.elders.len()),
```

Comment:
Don't think this is correct. This function is supposed to aggregate enough votes that if **this node votes** too, the vote  accumulates. But with this change it's going to accumulate regardless of this node voting.